site/es/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: "layouts/blog-post.njk"
+title: "Qué hay de nuevo en DevTools (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "Usa DevTools en tu idioma preferido, nuevos dispositivos Nest Hub, nueva insignia de consultas de contenedor CSS y más novedades."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GzZ1YTEeHlnhUrbPBzmr.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*Traducción cedida por [Carlos Caballero](https://carloscaballero.io).*
+
+{% include 'partials/devtools/es/banner.md' %}
+
+
+## Utiliza DevTools en tu idioma preferido {: #localized }
+
+Chrome DevTools soporta más de 80 idiomas, lo que permite que lo puedas utilizar en tu idioma preferido.
+
+Ve a [Configuración](/docs/devtools/customize/#settings), a continuación selecciona tu idioma preferido siguiente la ruta  **Configuración avanzada** > **Idioma** y finalmente, reinicia las DevTools.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/eozpCcjmnn7zwya9zXu6.png", alt="Cambia el idioma en Configuración > Preferencias", width="800", height="494" %}
+
+{# https://chromium.googlesource.com/chromium/src/+/58abfbcdddae27fb43c17f43dbcc197f2570b5a5 #}
+
+Chromium issue: [1163928](https://crbug.com/1163928)
+
+
+## Nest Hub en la lista de dispositivos {: #nest-hub }
+
+Ahora puedes simular las dimensiones físicas de Nest Hub y Nest Hub Max utilizando el [modo dispositivo](/docs/devtools/device-mode/).
+
+Clic [Toggle Device Toolbar](/docs/devtools/device-mode/#viewport) &nbsp; {% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="Toggle Device Toolbar", width="20", height="22" %} &nbsp;, selecciona Nest Hub o Nest Hub Max en la lista de dispositivos. 
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KytKWMiC4cbFfVUOBzlm.png", alt="Nest Hub en el modo dispositivo", width="800", height="549" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d13f911f7d98751cce659898936511b5ccda96cd #}
+
+Chromium issue: [1223525](https://crbug.com/1223525)
+
+
+## Ensayos de origen en la vista de detalle de Frame {: #origin-trials }
```

Comment:
In DevTools, isn't 'Origin Trials' translated as 'Pruebas de origen'? If so, could we please replace 'Ensayos de origen' with 'Pruebas de origen' to remain consistent?

Reply:
done!